Instance/application.make

```diff
@@ -265,6 +267,7 @@ $(APP_INFO_PLIST_FILE): $(GNUSTEP_STAMP_DEPEND) $(GNUSTEP_PLIST_DEPEND)
 	$(ECHO_CREATING)(echo "{"; echo '  NOTE = "Automatically generated, do not edit!";'; \
 	  echo "  NSExecutable = \"$(GNUSTEP_INSTANCE)\";"; \
 	  echo "  NSMainNibFile = \"$(MAIN_MODEL_FILE)\";"; \
+	  echo "  NSMainStoryboardFile = \"$(MAIN_STORYBOARD_FILE)\";"; \
```

Comment:
It would be nicer to only generate the entries that actually have a value, but as the old code wasn't testing this either this may be OK.

Reply:
Ok.